tests/phpunit/tests/rest-api/rest-server.php

```diff
@@ -2303,4 +2343,15 @@ public function data_envelope_params() {
 			array( array( 'alternate' ) ),
 		);
 	}
+
+	/**
+	 * Data provider.
+	 *
+	 * @throws Exception
```

Comment:
```suggestion
```
This shouldn't throw an exception.

Reply:
@costdev This comes from upstream method. See 
https://github.com/WordPress/wordpress-develop/blob/252a2c1f182653e3f3944eda0bbdbf49c0d72470/tests/phpunit/includes/abstract-testcase.php#L1151-L1171